samples/tfm/tfm_hello_world/src/main.c

```diff
@@ -74,18 +74,18 @@ void main(void)
 	printk("%s\n", hello_string);
 
 #if defined(CONFIG_TFM_PARTITION_PLATFORM)
-	uint32_t part;
+	uint32_t info_ram;
 
 	printk("Reading some secure memory that NS is allowed to read\n");
 
 	NRF_TIMER1_NS->TASKS_START = 1;
-	part = secure_read_word((intptr_t)&NRF_FICR_S->INFO.PART);
+	info_ram = secure_read_word((intptr_t)&NRF_FICR_S->INFO.RAM);
 	NRF_TIMER1_NS->TASKS_CAPTURE[0] = 1;
 	printk("Approximate IPC overhead us: %d\n", NRF_TIMER1_NS->CC[0]);
 
-	printk("FICR->INFO.PART: 0x%08x\n", part);
-	printk("FICR->INFO.VARIANT: 0x%08x\n",
-		secure_read_word((intptr_t)&NRF_FICR_S->INFO.VARIANT));
+	printk("FICR->INFO.RAM: 0x%08x\n", info_ram);
+	printk("FICR->INFO.FLASH: 0x%08x\n",
```

Comment:
The regex in sample.yaml needs to be changed accordingly. (I'm surprised this doesn't fail - is it not run in CI?)

Reply:
This does fail in CI